lib/tasks/archive.rake

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+ARCHIVE = Rails.env.test? ? Pathname(Dir.mktmpdir) : Rails.root.join('archive')
+
+namespace :roadeo do
+  desc 'Create a static archive of the Roadeo scoreboard'
+  task archive: [:environment, 'archive:index', 'archive:assets']
+
+  namespace :archive do
+    desc 'Archive the HTML scoreboard to `/archive/index.html`'
+    task index: ARCHIVE.join('index.html')
+
+    file ARCHIVE.join('index.html') => :environment do |t|
```

Comment:
I'm afraid to ask this question but I really don't think I'm going to find the answer myself, this time. It seems like this can only make a file called `index.html`. Wouldn't that preclude making one for every year? Do we just want one archive file ever?

Reply:
My idea was that the "archive" for a given year is a self-contained "site", yes.

You _can_ do otherwise, though; it just dumps everything in the `archive/` directory. Since you're going to have to move those files.... somewhere in order to host them as a website, you could also rename them in the process (even manually edit the HTML if you need to).